src/Illuminate/Support/composer.json

```diff
@@ -24,7 +24,8 @@
         "illuminate/contracts": "^10.0",
         "illuminate/macroable": "^10.0",
         "nesbot/carbon": "^2.62.1",
-        "voku/portable-ascii": "^2.0"
+        "voku/portable-ascii": "^2.0",
+        "rmunate/laravel_helpers": "^1.2"
```

Comment:
why are you requiring your package if it's not used in the PR at all?

also, if this is necessary, please alphabetize the entry.

Reply:
The command and use logic is already in the dependency, in the same way if you prefer, I pass the logic directly to the Framework.